Job summary can't find resources for JSON to MD

### Describe your question

Using the [Display the linting result in the GitHub Actions summary example](https://github.com/10up/wpcs-action/blob/develop/README.md#display-the-linting-result-in-the-github-actions-summary), I was able to successfully set up an action that tests my repository.

However, it looks to be failing with a `404 Not Found` error when trying to convert JSON to MD, at the Job Summary creation step.

Have I missed (or misunderstood) a step somewhere?


```
Run npx --yes github:10up/phpcs-json-to-md --path ./phpcs.json --output ./phpcs.md
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@10up%2fphpcs-json-to-md - Not found
npm ERR! 404 
npm ERR! 404  '@10up/phpcs-json-to-md@1.0.0' is not in this registry.
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2022-09-12T07_11_35_065Z-debug-0.log
Error: Process completed with exit code 1.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/README.md
+++ b/README.md
@@ -98,7 +98,8 @@ jobs:
             extra_args: '--report-json=./phpcs.json'
         - name: Update summary
           run: |
-            npx --yes github:10up/phpcs-json-to-md --path ./phpcs.json --output ./phpcs.md
+            npm i -g github:10up/phpcs-json-to-md
+            phpcs-json-to-md --path ./phpcs.json --output ./phpcs.md
             cat phpcs.md >> $GITHUB_STEP_SUMMARY
           if: always()
 ```

